db/migrate/20151110191852_create_redirections.rb

```diff
@@ -0,0 +1,10 @@
+class CreateRedirections < ActiveRecord::Migration
+  def change
+    create_table :redirections do |t|
+      t.timestamps null: false
+      t.references :next
```

Comment:
Yeah that's cool. Oh hmm, actually -- should we have uniqueness validations on url, and next so that two redirections don't have the same URL and don't point at the same place?